### PR TITLE
Create resource endpoint

### PR DESCRIPTION
**Pull request by rtichauerv**
Pretty standard endpoint creation :)

### CODE REVIEW

File: spec/integration/api_resources_controller_spec.rb [hunk 1]
```diff
@@ -38,4 +38,58 @@
...
+          id: { type: :integer },
+          name: { type: :string },
+          url: { type: :string }
+        }
+        let(:learning_unit_id) { create(:learning_unit).id }
+        let(:resource) { { name: 'test_resource', url: 'http://test.io' } }
+        run_test!
+      end
+
+      response '400', 'Invalid parameters' do
```
Comment (iaheck) [inline spec/integration/api_resources_controller_spec.rb:68]: if the URL's prefix is the only parameter that we are validating, I think that we need to be clearer on what's the reason of the error. At least on the documentation that we're creating.

I tried the API on a local environment via swagger and it was not intuitive that the default parameters (`{
  "name": "string",
  "url": "string"
}`) did not work properly and returned 400

Comment (iaheck) [inline spec/integration/api_resources_controller_spec.rb:68]: I also think that we should validate that name at least (and maybe URL as well) are present in the request body

**Review by iaheck — CHANGES_REQUESTED** [review]
looks flawless

File: spec/integration/api_resources_controller_spec.rb (hunk 1 above)
Comment (rtichauerv) [reply spec/integration/api_resources_controller_spec.rb:68]: done

**Review by iaheck — APPROVED** [review]

**Pull request merged**